tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
I tried to avoid creating new needles so `check_screen` seemed the best way to refactor this code. If it's not sufficient, I'll check it again next year.